core/cas-server-core-cookie-api/src/test/java/org/apereo/cas/web/support/CookieRetrievingCookieGeneratorTests.java

```diff
@@ -134,7 +134,6 @@ public void verifyCookieValueMissing() {
         val cookie = gen.retrieveCookieValue(request);
         assertNull(cookie);
     }
-
```

Comment:
Please restore the line break; These help with readability.

Reply:
The line break has been removed.